plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/PlayPublishPackageBase.kt

```diff
@@ -41,8 +44,10 @@ abstract class PlayPublishPackageBase : PlayPublishTaskBase() {
             versionCodes = versions
         }
 
-        track.releases = listOf(trackRelease)
-
+        val track = Track().apply {
```

Comment:
We're now always creating a new Track object instead of trying to update an existing one. Is that safe to do?

Reply:
Yeah: the track only has two fields and we overwrite both of them... So there was no point to what we were doing before. 😁

Reply:
Well for the `Track#track` field, we just found one that matched the `extension.track` which is the same as just setting it directly if you're going to overwrite the releases.

Reply:
Never mind, there's no other state than the list of releases and we're setting our own list anyway.

Reply:
Yup 👍